README.md

```diff
@@ -1,3 +1,5 @@
+/*
```

Comment:
If you could go ahead and make that a PR instead of a comment

![lumbergh](https://cloud.githubusercontent.com/assets/1264398/24562008/7ae146ec-1649-11e7-8c00-914f56632b61.jpg)

That'd be great.